### ISSUE TITLE
Use of `trim()`

### ISSUE THREAD

**Issue by jrean** [dusterio/link-preview#11]
Hi,
What do you think about adding a `trim()` for the `title` and `description` attributes?
Sometimes the string parsed has some extra white spaces before and after...

**Comment by dusterio** [comment]
Yes, I think it makes sense to trim these strings. I will add this
